src/styles/main.scss

```diff
@@ -0,0 +1,69 @@
+@import 'variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  width: $columns * ($day-size + $gap) + 2 * $pad;
+  gap: $gap;
+  padding: $pad;
+
+  &__day {
+    position: relative;
+    box-sizing: border-box;
+    display: flex;
+    flex: 0 0 $day-size;
+    height: $day-size;
+    background-color: $bg-color;
+    border: #{$gap} solid black;
```

Comment:
The border property is using interpolation for the gap variable. Make sure that $gap is defined and has a valid value in the 'variables' file.